packages/webchat/src/components/messages/MessageGroup.tsx

```diff
@@ -91,16 +89,12 @@ class MessageGroup extends React.Component<Props> {
             </span>
             {sortBy(messages, ['sent_on', 'eventId']).map((message, i, messages) => {
               const isLastMsg = i === messages.length - 1
-              const payload = this.convertPayloadFromOldFormat(message)
-
-              /*
-              // TODO: use the components package
-              if (payload?.wrapped) {
-                payload = { ...payload, wrapped: renderPayload(payload.wrapped) }
-              } else {
-                payload = renderPayload(payload)
+              let payload = this.convertPayloadFromOldFormat(message)
+
+              // TODO: remove this. quick_reply should be single-choice
+              if (payload.type === 'single-choice') {
```

Comment:
Any idea what is the root cause of this?

Reply:
We send a message of type 'single-choice' and when the user clicks on a button it returns a message of type quick_reply. So the components package simply uses the wrong name. I'll fix this problem in another PR